File: code-samples/webinar/quick-start.js
const RC = require('@ringcentral/sdk').SDK

var rcsdk = new RC({
    'server':       process.env.RC_SERVER_URL,
    'clientId':     process.env.RC_APP_CLIENT_ID,
    'clientSecret': process.env.RC_APP_CLIENT_SECRET
});
var platform = rcsdk.platform();
platform.login({ 'jwt':  process.env.RC_USER_JWT })

platform.on(platform.events.loginSuccess, function(e){
    create_webinar()
});

async function create_webinar(){
    try {
	platform.post('/webinar/configuration/v1/webinars', {
	    title: "My first webinar",
	    description: "This webinar was created via the Webinar Quick Start guide for developers"
	})
        .then(function(resp) {
          return resp.json()
        })
        .then(function (json) {
            console.log('Response: ', json )
        });
    } catch(e) {
        console.log("An error occured: ", e.message)
        process.exit(1)
    }
});

File: code-samples/webinar/quick-start.py
#!/usr/bin/python

# You get the environment parameters from your 
# application dashbord in your developer account 
# https://developers.ringcentral.com

import os
import sys
 
from dotenv import load_dotenv
from ringcentral import SDK
load_dotenv()
 
rcsdk = SDK( os.environ.get('RC_CLIENT_ID'),
             os.environ.get('RC_CLIENT_SECRET'),
             os.environ.get('RC_SERVER_URL') )
platform = rcsdk.platform()
try:
  platform.login( jwt=os.environ.get('RC_JWT') )

except Exception as e:
  sys.exit("Unable to authenticate to platform: " + str(e))

def create_webinar(fromNumber):
  try:
    resp = platform.post('/webinar/configuration/v1/webinars',
    {
	'title': "My first webinar",
	'description': "This webinar was created via the Webinar Quick Start guide for developers"
    })
    jsonObj = resp.json()
  except:
    sys.exit("Unable to create a webinar")
    print (jsonObj.messageStatus)

create_webinar()
